Makefile

```diff
@@ -1,21 +1,26 @@
-clean:
-	rm -rf build dist pymoo.egg-info
+.PHONY: clean
```

Comment:
Minor robustness change in case a file called `clean` existed; see e.g. [this post](https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile)